pkg/config/config.go

```diff
@@ -20,21 +20,24 @@ type Module struct {
 	ACL  []string `toml:"acl"`
 }
 
+func NewModule(name string, path string, acl []string) Module {
```

Comment:
This is needed for the client code that uses this as a library.

Since this is somewhat core concept of the serving part, I was left wondering if the module type should belong to root level package in this repo, or actually in rsyncd package, but I don't know. I wrote my question here mainly to flag it for you if you have any thoughts. Here is probably fine too.

Reply:
The generic name of this package brought a problem the very first time I imported into my concrete project. I had to rename the import:

```go
rsyncdconfig "github.com/gokrazy/rsync/pkg/config"
```

because my package already had declared `config` identifier. Should the package be renamed to `rsyncdconfig`, or do you think having the user rename the import is the better way?